SportHub.Business/Interfaces/IMainArticlesServices.cs

```diff
@@ -0,0 +1,10 @@
+using SportHub.Data.Entities;
+
+namespace SportHub.Business.Interfaces;
+
+public interface IMainArticlesServices
+{
+    Task<IEnumerable<MainArticle>> GetMainArticlesByLanguageIdAsync(string languageId);
+    Task CreateMainArticlesAsync(IEnumerable<MainArticle> mainArticles);
+    Task<IEnumerable<ArticleForMainArticleForm>> GetMainArticlesWithAdditionalInformationByLanguageIdAsync(string languageId);
```

Comment:
Name of Entity and of method is totally different. I would prefer something like GetMainArticlesWithDetails